src/components/InputTooltip.js

```diff
@@ -15,13 +15,13 @@ class InputTooltip extends Component {
     }
 
     render() {
-        const { onAccept, onRemove, isLinkInvalid } = this.props;
+        const { onAccept, onRemove, isLinkInvalid, linkValue } = this.props;
 
         return (
             <div className="pe-input-tooltip">
                 <div className="wds-input">
                     <div className="wds-input__field-wrapper">
-                        <input placeholder="URL" className="wds-input__field" ref={(el) => { this.input = el; }} />
+                        <input placeholder="URL" className="wds-input__field" ref={(el) => { this.input = el; }} value={linkValue} />
```

Comment:
shouldn't we use `defaultValue` prop here?
https://reactjs.org/docs/uncontrolled-components.html

Reply:
no, because we want to overwrite the input value with the one from the current selection